common/datasets/librispeech/cart.py

```diff
@@ -0,0 +1,939 @@
+                                "values": "EH ER AH AO",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-CLOSEFULL",
+                                "values": "IY UW",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-CLOSENEAR",
+                                "values": "IH UH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-UNROUNDED",
+                                "values": "IY EH AE IH ER AH AA",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-ROUNDED",
+                                "values": "UH UW AO",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-FRONT",
+                                "values": "IY EH AE IH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-BACK",
+                                "values": "UW UH AH AO AA",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-SAMPA-a",
+                                "values": "AW AY",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-SAMPA-U",
+                                "values": "UH AW OW",
+                            },
+                            {
+                                "type": "question",
+                                "description": "VOWEL-SAMPA-I",
+                                "values": "IH AY EY OY",
+                            },
+                            {
+                                "type": "question",
+                                "description": "fricat",
+                                "values": "F V TH S Z SH ZH HH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "voiced",
+                                "values": "B D G V Z M N NG L R W JH TH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "voiceless",
+                                "values": "P T K F S SH CH HH",
+                            },
```

Comment:
could we sort these questions further above to the other consonant questions?

I don't think it'll make much of a difference, just for my sense of order